openslides_backend/services/authentication.py

```diff
@@ -50,9 +50,8 @@ def get_user(self, headers: Headers) -> int:
             try:
                 user_id = body["user_id"]
             except (TypeError, KeyError):
-                raise AuthenticationException(
-                    "Empty or bad response from authentication service."
-                )
+                # TODO: remove this and throw an AuthentificationException when permissions are implemented.
+                user_id = 1
```

Comment:
To not be dependent on the right communication between auth and backend and since at first, we want to build the app admin-only for the first milestone, I added this.

Reply:
Can you move this up: Just do not do an request. We have to wait a bit for the auth service.

Reply:
Done